src/Platforms/MariaDb1060Platform.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Doctrine\DBAL\Platforms;
+
+/**
+ * Provides the behavior, features and SQL dialect of the MariaDB 10.6 (10.6.0 GA) database platform.
+ *
+ * Note: Should not be used with versions prior to 10.6.0.
+ */
+class MariaDb1060Platform extends MariaDb1052Platform
```

Comment:
This new platform class is not being used anywhere.

Reply:
Addd it to `\Doctrine\DBAL\Driver\AbstractMySQLDriver::createDatabasePlatformForVersion`